### PR TITLE
Support PDF

### PR DESCRIPTION
**Pull request by codler**

### CODE REVIEW

**Review by JohnLindahlTech — COMMENTED** [review]
LGTM, lets release a beta, since tests and documentation is missing.

File: src/index.js [hunk 1]
```diff
@@ -178,6 +182,10 @@ function isJSON(contentType) {
   return contains('application/json')(contentType);
 }
 
+function isPDF(contentType) {
+  return contains('application/pdf')(contentType);
+}
```
Comment (iamstarkov) [inline src/index.js]: i dont understand how does it work. can help me, @codler ?

Comment (codler) [reply src/index.js]: When the content-type is application/pdf our http response are neither a json or plain text, the response are a binary file so fetch api need to call the method blob

Comment (iamstarkov) [reply src/index.js]: then let's call it blob and `isBLOB`

**Review by iamstarkov — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (iamstarkov) [reply src/index.js]: got it, thanks.
pls, rename and we will ship it

**Comment by codler** [comment]
Its renamed now

**Comment by iamstarkov** [comment]
@JohnPhoto dont merge it now, lets talk tomorrow and merge it after

**Comment by iamstarkov** [comment]
my point its a regular patch semver pull-request, no need in beta. merge it without that version bump and release it as regular patch

**Pull request merged**